css_files/question_page.css

```diff
@@ -1,12 +1,12 @@
 @import "./home_page.css";
 
-.question-container{
+.question-container,.rules-container{
     display: grid;
     place-items: center;
     height: 60vh;
 }
 
-.question{
+.question,.rules{
     min-width: fit-content;
```

Comment:
The min-width is by default `fit-content`, you didn't need to specify it

Reply:
Ohh okay, I will remove it.